test/Datadog.Trace.Tests/Propagators/B3SpanContextPropagatorTests.cs

```diff
@@ -185,18 +184,5 @@ private static IHeadersCollection InjectContext(IHeadersCollection headers, stri
 
             return headers;
         }
-
-        private static void AssertExpected(IHeadersCollection headers, string key, string expected)
-        {
-            var matches = headers.GetValues(key);
-            Assert.Single(matches);
-            matches.ToList().ForEach(x => Assert.Equal(expected, x));
-        }
-
-        private static void AssertMissing(IHeadersCollection headers, string key)
-        {
-            var matches = headers.GetValues(key);
-            Assert.Empty(matches);
-        }
```

Comment:
why these methods have been moved?

Reply:
Reused for CompositeTextMapPropagatorTests, HeadersCollectionTestBase seems to be ideal for tests that deal with headers.

Reply:
I see. Only `AssertMissing` is not reused there but I think we can keep it as you refactored 👍

Reply:
Yep, both seemed to be generic / reusable. Seems bit weird to leave one out 😄